### ISSUE TITLE
ошибка работы расписания

### ISSUE THREAD

**Issue by Banzaizaizai** [tretyakovsa/Sonoff_WiFi_switch#107]
![2018-11-23](https://user-images.githubusercontent.com/13425455/48959854-1821a080-ef71-11e8-92c7-f6d78824b2e0.png)
скрин с расписанием

![2018-11-23](https://user-images.githubusercontent.com/13425455/48959879-44d5b800-ef71-11e8-9311-3bc40616975c.png)

подключенные реле

при наступлении времени выполнения - отрабатывает только 2-ое реле. первое нет.

**Comment by renat2985** [comment]
Скорее всего причина в том что у вас на одно и тоже время несколько таймеров. Попробуйте допустим второе реле включать и выключать на несколько секунд позже или раньше.

**Comment by tretyakovsa** [comment]
Да группу действий пока сделать нельзя. Это можно будет сделать через макрос. Макросы сейчас работают из сценариев. Но выполнения из таймеров еще не готово.

**Comment by Banzaizaizai** [comment]
да. все правильно.  если разнести на 10 секунд - то отрабатывает.

**Comment by renat2985** [comment]
Принцип работы таймеров примерно в следующем. ESP у нас не отслеживает все таймеры. Отслеживается только 1 таймер ближайший по времени к выполнению. Как только он отработает в переменную добавляется второй таймер который по очереди ближе к  выполнению. И так далее. Это позволяет делать неограниченное количество таймеров и не загружать ресурсы ESP.
Но если вы делаете 2 таймера на 1 и тоже время, второй скорее всего не отработает. Поэтому их нужно делать разными хотя бы на несколько секунд.
